exercises/typescript/README.md

```diff
@@ -0,0 +1,13 @@
+## Setting Up
+
+You will need npm and node v14.15.0 or later. There are many different ways to install it https://nodejs.dev/learn/how-to-install-nodejs
+
+Windows setup instructions are here : https://phoenixnap.com/kb/install-node-js-npm-on-windows
+
+## Running tests
+
+On the command line, navigate to this directory (`cd exercises/javascript`) and run `npm install` followed by `npm test`.
```

Comment:
I assume the path should be 'typescript':
```suggestion
On the command line, navigate to this directory (`cd exercises/typescript`) and run `npm install` followed by `npm test`.
```

Reply:
Good spot, I haven’t updated the readme yet but I also need to add in some more info about how to run each class.